tests/foreman/cli/test_org.py

```diff
@@ -16,6 +16,8 @@
     data, run_only_on, skip_if_bug_open, stubbed)
 from robottelo.test import CLITestCase
 
+import random
```

Comment:
nitpick: move the `import` statement lines before `from ... import ...` lines and separate the blocks by one blank line.